### DISCUSSION TITLE
How to support other databases, such as Dameng?

### DISCUSSION THREAD

**Discussion by Lce-Lemon** [prisma/prisma#24165] [Q&A]
### Question

See the title

### Information about Prisma Schema, Client Queries and Environment (optional)

```prisma
// Add your schema.prisma
```

```ts
// Add any relevant Prisma Client queries here
```

- OS: 
- Database: 
- Node.js version: 

- Run `prisma -v` to see your Prisma version and paste it

**Comment by ludralph** [comment]
Hi @Lce-Lemon 👋 

Thank you for raising this question.

To see support for Dameng database, there will have to be a connector for it that is compatible with Prisma ORM. You can open a [feature request](https://github.com/prisma/prisma/issues/new?assignees=&labels=&template=feature_request.md&title=) for supporting Dameng database so that our engineers can investigate this further.

If this answers your question, it would be great if you could **mark this Discussion as answered** to indicate that it has been resolved. 

Otherwise please let us know how else we can help you further or close the Discussion if it was resolved in some other way 🙏

**Reply by Lce-Lemon** [reply]
Does Prisma have an extension mechanism to support other databases?

**Reply by ludralph** [reply]
Hi @Lce-Lemon 

At the moment no, but you can read how we are thinking about this [here](https://github.com/prisma/prisma/discussions/25772#discussioncomment-11853585).

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that haven’t had any recent activity and appear to be outdated.

If this discussion is still important to you or unresolved, we’d love to hear from you! Feel free to reopen it or start a new one with updated details.

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and being part of the community!;
